conf/application.conf

```diff
@@ -46,7 +46,7 @@ play.http.errorHandler = "uk.gov.hmrc.play.bootstrap.backend.http.JsonErrorHandl
 # ~~~~
 # Additional play modules can be added here
 
-play.modules.enabled += "uk.gov.hmrc.mongo.play.PlayMongoModule"
+#play.modules.enabled += "uk.gov.hmrc.mongo.play.PlayMongoModule"
```

Comment:
is this supposed to be commented?

Reply:
we're not using mongo atm so I've commented it out for now - I haven't stuck the mongo config in the app-configs so it might break in the environments if we uncomment it out. We could delete it, but if we need it in the future it'd just be easier to uncomment it out